### ISSUE TITLE
Include instructions to exclude Best-practice and Experimental failures from axe-core scans

### ISSUE THREAD

**Issue by adarshrema** [microsoft/axe-pipelines-samples#27]
Add instructions on how to restrict axe scans to include only failures corresponding to WCAG 2.0 A and AA rules:

use the AxeBuilder.withTags method as follows when you initialize Axe in your test code:

```js
AxeBuilder(driver)
  .withTags(['wcag2a', 'wcag2aa'])
```

**Comment by ferBonnin** [comment]
since this is closed marking it as resolved
